smudge.el

```diff
@@ -4,7 +4,7 @@
 
 ;; Keywords: multimedia, music, spotify, smudge
 ;; Package: smudge
-;; Package-Requires: ((emacs "24.4") (simple-httpd "1.5") (request "0.3"))
+;; Package-Requires: ((emacs "27.1") (simple-httpd "1.5") (request "0.3") (oauth2 "0.16)
```

Comment:
Funny - I had that and `package-lint` suggested I remove it since it's not installable. I guess it's better to have it after all!